packages/keystone/fetch.js

```diff
@@ -146,20 +145,15 @@ const fetchWithRetriesAndLogger = async (url, options = {}) => {
     // At least one request on maxRetries = 0
     do {
         try {
-            const controller = new AbortController()
-            const signal = controller.signal
-            const response = await Promise.race([
-                fetchWithLogger(url, { ... fetchOptions, signal }, { skipTracingHeaders, skipXTargetHeader }),
-                new Promise((_, reject) =>
-                    setTimeout(() => {
-                        controller.abort()
-                        reject(new Error('Abort request by timeout'))
-                    }, abortRequestTimeout)
-                ),
-            ])
+            const response = await fetchWithLogger(url, {
+                ...fetchOptions,
+                signal: AbortSignal.timeout(abortRequestTimeout),
```

Comment:
Is it working?
<img width="799" alt="image" src="https://github.com/user-attachments/assets/2ecf0cf4-a92d-49e9-a3a1-fc41fe682f9a">

Reply:
unfortunately yes
<img width="432" alt="image" src="https://github.com/user-attachments/assets/8c6112a0-656d-46be-a1c9-be3dfe2b927a">